arroyo-console/src/routes/connections/CreateConnection.tsx

```diff
@@ -88,6 +159,57 @@ function ConfigureKafka({
         <Input type="text" value={config.bootstrapServers} onChange={onChange} />
         <FormHelperText>Comma-separated list of kafka brokers to connect to</FormHelperText>
       </FormControl>
+      <FormControl isRequired>
+  <FormLabel>Authentication Type</FormLabel>
+  <Select value={config.authConfig?.authType.case || ""} onChange={onChangeAuthType}>
+    <option value="">Select an authentication type</option>
+    <option value="noAuth">No Auth</option>
+    <option value="saslAuth">SASL Auth</option>
+  </Select>
+  {config.authConfig?.authType.case === "saslAuth" && (
+  <Stack spacing={5}>
```

Comment:
This should be indented a level so that it's clear it's part of the conditional

Reply:
This is also incorrectly nested under the FormControl; a FormControl should have only one child (visually this leads to the spacing being off)

Reply:
I think I fixed this.